statviz/src/utils/hooks/useCreatedBoxes.ts

```diff
@@ -0,0 +1,35 @@
+import { gql, useQuery } from "@apollo/client";
+import {
+  CreatedBoxesData,
+  CreatedBoxesResult,
+} from "../../types/generated/graphql";
+import { useMemo } from "react";
+import { createdBoxesTable } from "../table";
+import { useSearchParams } from "react-router-dom";
+
+export default function useCreatedBoxes(data?: {
+  createdBoxes: CreatedBoxesData;
+}) {
+  const [searchParams] = useSearchParams();
```

Comment:
maybe move data querying in here. see https://github.com/boxwise/boxtribute/blob/master/react/src/hooks/useMoveBoxes.ts